browser-test/src/support/admin_questions.ts

```diff
@@ -423,6 +455,19 @@ export class AdminQuestions {
     questionText = 'radio button question text',
     helpText = 'radio button question help text',
     enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION) {
+    await this.createRadioButtonQuestion(questionName, options, description, questionText, helpText, enumeratorName);
+
+    await this.expectAdminQuestionsPageWithCreateSuccessToast();
+
+    await this.expectDraftQuestionExist(questionName, questionText);
+  }
+
+  async createRadioButtonQuestion(questionName: string,
+    options: Array<string>,
+    description = 'radio button description',
+    questionText = 'radio button question text',
+    helpText = 'radio button question help text',
+    enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION){

```

Comment:
suggestion (non-blocking): this is a lot of params, it may be easier to read at the call site if they were bundled in an object

```suggestion
type CreateRadioButtonQuestionParams = {
  questionName: string;
  options: Array<string>;
  description?: string;
  questionText?: string;
  helpText?: string;
  enumeratorName?: string;
}

  async createRadioButtonQuestion({
    questionName,
    options,
    description = 'radio button description',
    questionText = 'radio button question text',
    helpText = 'radio button question help text',
    enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION}: CreateRadioButtonQuestionParams) {
```

Reply:
Updated - let me know what you think! Can also change the add__Question functions to follow this pattern as a follow up.